org.buhocms.BuhoCMS.yml

```diff
@@ -0,0 +1,33 @@
+app-id: org.buhocms.BuhoCMS
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: buhocms
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=host
+  - --talk-name=org.freedesktop.Flatpak
+modules:
+  - name: BuhoCMS
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - './build.sh'
+    sources:
+      - type: file
+        # path: ./BuhoCMS-Linux-Portable.tar.gz
+        url: https://github.com/iakmds/buhocms/releases/download/v0.2.0/BuhoCMS-Linux-Portable.tar.gz
```

Comment:
let met guess. This is an x86_64 binary (ie not portable).

But you still build an aarch64 flatpak.

Reply:
I believe you are right. What do you advise to do?

Reply:
I limited the architecture to x86_64 in https://github.com/flathub/flathub/pull/3939/commits/afca2ec8189b5d3efd90aa10fc61159ac979f5ee